pygmt/tests/test_session_management.py

```diff
@@ -43,10 +44,12 @@ def test_gmt_compat_6_is_applied(capsys):
             )
             assert err == ""  # double check that there are no other errors
     finally:
-        with Session() as lib:
-            # revert gmt.conf back to GMT_COMPATIBILITY = 6
-            lib.call_module("set", "GMT_COMPATIBILITY 6")
         end()
-        begin()  # Restart the global session
+        # Clean up the global "gmt.conf" in the current directory
+        assert os.path.exists("gmt.conf")
+        os.remove("gmt.conf")
```

Comment:
Have to remove the "gmt.conf" in the current directory so that it won't affect other tests.